go/protocol/keybase1/extras.go

```diff
@@ -418,6 +418,20 @@ func MakeTestTeamID(n uint32) TeamID {
 	return tid
 }
 
+// Used by unit tests.
+func MakeTestSubTeamID(n uint32) TeamID {
+	b := make([]byte, 8)
```

Comment:
@songgao: nope.  This is the same as the two other test ID functions above.  It just adds some zeroes into the ID.